src/pages/Home.tsx

```diff
@@ -1,12 +1,23 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { DragDropContext, DropResult } from "react-beautiful-dnd";
 import { tasks as initialTasks, Task } from "../data/tasks";
 import TaskList from "../components/TaskList/TaskList";
 import TaskForm from "../components/TaskForm/TaskForm";
 
 const Home: React.FC = () => {
-  const [taskList, setTaskList] = useState<Task[]>(initialTasks);
+  // Initialiser avec les tâches du localStorage s'il y en a, sinon utiliser initialTasks
+  const [taskList, setTaskList] = useState<Task[]>(() => {
+    const savedTasks = localStorage.getItem('tasks');
+    return savedTasks ? JSON.parse(savedTasks) : initialTasks;
+  });
   const [showPopup, setShowPopup] = useState(false);
+  const [selectedTask, setSelectedTask] = useState<Task | null>(null);
+
+  // Sauvegarder dans localStorage à chaque modification
+  useEffect(() => {
```

Comment:
for this project, I prefer that you don't use local storage. so this useEffect is unnesecary.